tests/Admin/BaseMediaAdminTest.php

```diff
@@ -29,10 +29,29 @@
 
 class BaseMediaAdminTest extends TestCase
 {
+    /**
+     * @var ObjectProphecy|Pool
```

Comment:
```suggestion
     * @var ObjectProphecy&Pool
```

The mock will provide both methods